store/event_test.go

```diff
@@ -84,6 +88,15 @@ func TestScanHistory(t *testing.T) {
 	if e != nil {
 		t.Fatalf("bad index shoud reuturn nil")
 	}
+
+	// Add a dir which will replace old event
```

Comment:
oh. i see what you are doing. you changed the history size to 6. if you want to test the compact behavior, can you please create a new test for it? this test is for testing the actual event scanning.